README.md

```diff
@@ -168,6 +168,8 @@ yarn build # to build the project
 Afterwards, you can find the build output in the `dist` folder of the corresponding package folder.
 For example, to find the Button component (that belongs to the `main` package), look inside the `packages/main/dist` folder.
 
+*Note: before building the project you can also set the `PUBLIC_DEPLOY_PATH` environment variable to specify the path where non-bundled assets will be fetched from*
+
```

Comment:
It might be good to add the example from the commit description in the docs
`DEPLOY_PUBLIC_PATH="\/my\/resources\/" yarn build`

note however that this gives the wrong result on mac (when reading from nodejs)
`DEPLOY_PUBLIC_PATH: '\\/my\\/resources\\/'`